lib/mayaUsd/fileio/writeJobContext.cpp

```diff
@@ -120,9 +120,21 @@ bool UsdMayaWriteJobContext::IsMergedTransform(const MDagPath& path) const
 
     // Any transform with multiple (non-intermediate) shapes below is
     // non-mergeable.
-    unsigned int numberOfShapesDirectlyBelow = 0u;
-    path.numberOfShapesDirectlyBelow(numberOfShapesDirectlyBelow);
```

Comment:
I translated the code of numberOfShapesDirectlyBelow in OpenMaya code and stop if there is more than one shape instead of going through all the shapes to count them since it is not necessary saving 1/2 of the export time,

Reply:
Thanks Yves.  Can you add in a comment saying that you expanded out the implementation of MDagPath::numberOfShapesDirectlyBelow() to cut down the time by half.  Otherwise, someone will come in and replace your code with numberOfShapesDirectlyBelow().